src/content/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-errors-network-failure-analysis.mdx

```diff
@@ -178,11 +176,11 @@ Error profiles provide visual details about significant differences in the frequ
 
 This helps you take more of the guesswork out of resolving your mobile application's HTTP errors. You can more easily determine if you safely ignore the error, or if you should attempt to resolve the error with a new deployment, code change, customer communication or other actions.
 
-## View more details about a specific error [#error-summary]
+## Drill down into a single error [#error-summary]
```

Comment:
"drill down" seems like an idiom or phrasal verb that might not translate well to other languages or might not be understood by non-native english speakers.

Reply:
I need to get this PR out and haven't heard back from the translations team yet, so I'll merge, but if they object I'll do a find/replace of any of my mentions of "drill down". Thanks for raising a potential issue with this phrase - I've always been curious if other people use it.